examples/dice/instrumented/go.mod

```diff
@@ -26,6 +26,6 @@ require (
 )
 
 replace (
-	go.opentelemetry.io/contrib/bridges/otelslog => ../../bridges/otelslog
-	go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp => ../../instrumentation/net/http/otelhttp
+	go.opentelemetry.io/contrib/bridges/otelslog => ../../../bridges/otelslog
+	go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp => ../../../instrumentation/net/http/otelhttp
```

Comment:
question for @open-telemetry/go-approvers, would it be possible here to **not** take a dependency within the repository but as if this would be in an independent project folder? This way it would be easier to re-use and also pin the version of dependencies

Reply:
If this helps, the code-excerpt directives can be used to replace any part of the `../../../` path with something else. I know that it will create a separation between the actual code, but if it can help us get the OTel website code updated sooner, I'd suggest that we do it.

Reply:
This way we would have the examples not synced with the code in the repository.
People browsing a tag (e.g. https://github.com/open-telemetry/opentelemetry-go/tree/v1.32.0) would see examples for the previous version of OTel Go (e.g. `v1.31.0`).

A git tag is "release" in Go. Maybe you would simply prefer to keep this example in https://github.com/open-telemetry/opentelemetry.io and ask us for updating it if necessary after each new version of OTel Go is released?

Reply:
I don't want to copy the example into the opentelemetry.io repository, that's the whole purpose of this exercise to keep the code in control of the language SIG (and with that checked and verified) and not have a drift. We take care of that with the code excerpts, that's also why we need the instrumented and uninstrumented versions: we can not host 11 language-specific CI checks on the code on the website repo.

Thinking a little bit about it I argue that the `go.mod` file is not even that important since we ask people to do the go mod init and go get anyhow

Reply:
> Thinking a little bit about it I argue that the go.mod file is not even that important since we ask people to do the go mod init and go get anyhow.

Makes sense. Running `go mod tidy` should get all necessary dependencies without need of running `go get`.

Reply:
then I think I can resolve this conversation